modules/selection/main/index.coffee

```diff
@@ -16,9 +16,34 @@ getMethod = (node, methodName) ->
   else
     Element.prototype[methodName]
 
+
+# Should only be called with Function.call(node, selector)
+matchPolyfill = (selector) ->
+  node = this
+  matchingNodes = (node.document or node.ownerDocument).querySelectorAll(selector)
+  [].slice.call(matchingNodes).indexOf(node) > -1
+
+getMatches = (node) ->
+  node.matches or hx.vendor(node, 'matchesSelector') or matchPolyfill
```

Comment:
Uses hx.vendor - This is assuming that we don't need to do the 'getMethod' style of call?